LICENSE

```diff
@@ -1,7 +1,14 @@
-Copyright 2019 Martmists
+Copyright 2020 FabLabsMC
```

Comment:
No.

Reply:
Not sure we can keep this in FabLabs, then.